apps/shared/avifutil.c

```diff
@@ -220,6 +224,37 @@ avifAppFileFormat avifGuessFileFormat(const char * filename)
     return AVIF_APP_FILE_FORMAT_UNKNOWN;
 }
 
+avifAppFileFormat avifReadImage(const char * filename,
```

Comment:
Is it possible to modify the `avifInputReadImage` function in avifenc.c to call this new function?

Reply:
Done